distribution/galleon-feature-packs/adapter-galleon-pack/pom.xml

```diff
@@ -0,0 +1,318 @@
+                                    <includes>
+                                        <include>content/**</include>
+                                        <include>modules/**</include>
+                                    </includes>
+                                </resource>
+                            </resources>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.wildfly.galleon-plugins</groupId>
+                <artifactId>wildfly-galleon-maven-plugin</artifactId>
+                <version>${version.org.wildfly.galleon-plugins}</version>
+                <dependencies>
+                    <!--
+                        feature-spec-gen uses wildfly-embedded to generate the feature specs, hence the designated
+                        wildfly-embedded version must match the pack one
+                    -->
+                    <dependency>
+                        <groupId>org.wildfly.core</groupId>
+                        <artifactId>wildfly-embedded</artifactId>
+                        <version>${version.org.wildfly.core}</version>
+                    </dependency>
+                    <!-- If you add a dependency on wildfly-embedded you need to bring your own transitives -->
+                    <dependency>
+                        <groupId>org.wildfly.common</groupId>
+                        <artifactId>wildfly-common</artifactId>
+                        <version>${version.org.wildfly.common}</version>
+                    </dependency>
+                </dependencies>
+                <executions>
+                    <execution>
+                        <id>keycloak-adapter-galleon-pack-build</id>
+                        <goals>
+                            <goal>build-feature-pack</goal>
+                        </goals>
+                        <phase>compile</phase>
+                        <configuration>
+                            <fork-embedded>false</fork-embedded>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.keycloak</groupId>
+                <artifactId>keycloak-distribution-licenses-maven-plugin</artifactId>
+            </plugin>
+        </plugins>
+    </build>
+
+    <profiles>
+        <profile>
+            <id>community</id>
+            <activation>
+                <property>
+                    <name>!product</name>
+                </property>
+            </activation>
+            <properties>
+                <galleon-adapter-group-id>org.keycloak</galleon-adapter-group-id>
+                <version.org.wildfly>20.0.0.Final</version.org.wildfly>
+                <version.org.wildfly.core>12.0.1.Final</version.org.wildfly.core>
+                <version.org.wildfly.common>1.5.4.Final</version.org.wildfly.common>
```

Comment:
Version properties here should be aligned with already used properties in keycloak-parent pom.xml.
I have sent PR to your product repo fork.